cordapp-upgrades/build.gradle

```diff
@@ -0,0 +1,150 @@
+apply plugin: 'kotlin'
+apply plugin: 'net.corda.plugins.cordapp'
+apply plugin: 'net.corda.plugins.cordformation'
+
+buildscript {
+    ext {
+        corda_release_group = 'net.corda'
+        corda_release_version = '5.0-SNAPSHOT'  //TODO: set to Corda 4.1 when this is released
+        corda_gradle_plugins_version = '4.0.42'
+        kotlin_version = '1.2.71'
+        junit_version = '4.12'
+        quasar_version = '0.7.10'
+        log4j_version = '2.11.2'
+        jopt_simple_version = '5.0.2'
+        
+        corda_platform_version = '4'
+        
+        v1_contract = ":contracts:v1-contract"
+        v2_contract = ":contracts:v2-can-default"
```

Comment:
what does this stand for "v2-can-default" ?

Reply:
Good question - the extra bit I added to the contract is the ability to default on the obligation (although in practice this doesn't actually do anything other than set the field at the moment). The name is possibly not that clear though. Perhaps it would be better to just indicate there's a new field (call it something like "updated-contract-state")?

Reply:
Ah, ok. It was literally "Can default" .
It makes sense then. 
Maybe these names to be more explicit:

- "v1-basic-obligation"
- "v2-obligation-can-default"